share_extension/android/extension_channels/extension_channels.js

```diff
@@ -45,10 +52,12 @@ export default class ExtensionTeam extends PureComponent {
 
     state = {
         sections: null,
+        loading: true,
     };
 
     componentDidMount() {
         this.buildSections();
```

Comment:
I'm not 100% sure, but won't `buildSections` get called from within the new call to `searchChannels('')` ?

Wondering why this initialization call to `buildSections` is still necessary.